http.go

```diff
@@ -0,0 +1,68 @@
+package discordgo
+
+import (
+	"crypto/ed25519"
+	"encoding/hex"
+	"encoding/json"
+	"io"
+	"log"
+	"net/http"
+)
+
+type WebhookServer struct {
+	sess      *Session
+	publicKey ed25519.PublicKey
+	handlers  map[string]WebhookHandler
+}
+
+type WebhookHandler func(*Session, *Interaction)
```

Comment:
Yet again, confusing naming. Aside from that, I think would be better to make the second parameter `*InteractionCreate`, cause that's a standard for events, and I think it would be good to keep it here as well.